pvactools/lib/aggregate_all_epitopes.py

```diff
@@ -211,9 +217,12 @@ def get_list_unique_mutation_keys(self):
         return sorted(keys)
 
     def calculate_clonal_vaf(self):
-        vafs = np.sort(pd.read_csv(self.input_file, delimiter="\t", usecols=["Tumor DNA VAF"])['Tumor DNA VAF'].unique())[::-1]
-        vaf_clonal = list(filter(lambda vaf: vaf < 0.6, vafs))[0]
-        return vaf_clonal
+        if self.tumor_purity:
+            return self.tumor_purity * 0.5
+        else:
+            vafs = np.sort(pd.read_csv(self.input_file, delimiter="\t", usecols=["Tumor DNA VAF"])['Tumor DNA VAF'].unique())[::-1]
```

Comment:
A comment here might be nice to explain it.  
#if no tumor purity is provided, make a rough estimate by taking the list of VAFs < 0.6 (assumption is that these are CN-neutral) and return the largest as the marker of the founding clone